src/python/pants/backend/python/packaging/pyoxidizer/rules_integration_test.py

```diff
@@ -9,8 +9,32 @@
 
 
 def test_end_to_end() -> None:
```

Comment:
Btw this test is _really_ slow to run, a whole minute on my M1. Hence bundling a couple different things together.